src/containers/orgs/OrgsReducer.js

```diff
@@ -87,6 +102,27 @@ export default function orgsReducer(state :Map<*, *> = INITIAL_STATE, action :Ob
       });
     }
 
+    case addMemberToOrganization.case(action.type): {
+      const seqAction :SequenceAction = action;
+      return addMemberToOrganization.reducer(state, action, {
+        REQUEST: () => state
+          .setIn([ADD_MEMBER_TO_ORGANIZATION, 'requestState'], RequestStates.PENDING)
+          .setIn([ADD_MEMBER_TO_ORGANIZATION, seqAction.id], seqAction),
+        SUCCESS: () => {
+          const storedSeqAction :SequenceAction = state.getIn([ADD_MEMBER_TO_ORGANIZATION, seqAction.id]);
+          if (storedSeqAction) {
+            const { memberId } = storedSeqAction.value;
+            return state
+              .deleteIn(['memberSearchResults', memberId])
+              .setIn([ADD_MEMBER_TO_ORGANIZATION, 'requestState'], RequestStates.SUCCESS);
```

Comment:
you seem to update org members after removing one but not after adding one. is there any reason for that?

Reply:
Yes.

To add a member, I first have to search via the `PrincipalsApi`. What I get back as member search result is a different type than what is stored in redux. What I do instead is reload the org details on successful add:

```
if (requestStates[ADD_MEMBER_TO_ORGANIZATION] === RequestStates.SUCCESS
    && prevProps.requestStates[ADD_MEMBER_TO_ORGANIZATION] === RequestStates.PENDING) {
  actions.getOrganizationDetails(orgId);
}
```

Reply:
oooh i see. that does make sense!